app/src/hooks/AuthHandler.js

```diff
@@ -38,6 +40,15 @@ const AuthHandler = (onComplete) => {
         window.uid = authData?.uid;
         localStorage.setItem("__rq_uid", authData?.uid);
 
+        if (user.metadata.creationTime === user.metadata.lastSignInTime) {
```

Comment:
Also user will be null when logging out. so either add optional chaining or a null check to avoid `null.metadata`